docs/reference/enchantments.md

```diff
@@ -7,7 +7,6 @@ To find out how to apply enchantments, click [here](../config/crate#examples)
 
 | Name                     | Alternate Name           |
 | ------------------------ | ------------------------ |
-| unbreaking               | DURABILITY               |
```

Comment:
Did we want to change the alternate name to durability?

Reply:
We had to use "durability", as "unbreaking" wouldn't work previously. But now durability conflicts with the durability tag for item damage.